examples/huggingface-transformers/README.md

```diff
@@ -0,0 +1,130 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Hugging Face Transformers - DeepSparse Inference Examples
+This directory contains examples for serving, benchmarking, and running NLP 
+models from the [Hugging Face Transformers](https://github.com/huggingface/transformers)
+repository using the DeepSparse Engine. These examples can load pre-trained,
+sparsified models from the [SparseZoo](https://github.com/neuralmagic/sparsezoo) 
+or you can specify your own transformer [ONNX](https://onnx.ai/) file.
+
+## Installation
+The dependencies for this example can be installed using `pip` and the supplied `requirements.txt` file:
+```bash
+pip3 install -r requirements.txt
+```
+## DeepSparse Pipeline Example
+
+The DeepSparse-Hugging Face pipeline integration provides a simple API 
+dedicated to several tasks,
+following is an example using a pruned BERT model from the SparseZoo for 
+Question-Answering task. The current version of the pipeline supports only 
+`question-answering` tasks, 
+
+```python
+from deepsparse.transformers import pipeline
```

Comment:
our `pipeline` here can also infer a default sparse model to run on the system, so we could also demonstrate a bare-bones instantiation of pipeline, just like in the [transformers README](https://github.com/huggingface/transformers)

```python
question_answerer = pipeline('question-answering')
```